netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
looks like netctl will only return netmaster's version and not netplugin's version or the docker/k8s plugin's version. 

I am thinking about the usefulness of being able to print partial info here. Should we just return netctl's version instead? Sorry if I am missing a usecase this will solve.